worker/src/handlers/schedule.ts

```diff
@@ -133,6 +135,7 @@ async function updateHistory(sentry: Toucan): Promise<void> {
     timestamp: timestampString,
     active_installations,
     installation_types: data.installation_types,
+    versions: grouped_versions,
```

Comment:
```suggestion
    versions: groupVersions(data.versions),
```
No need to assign this, it's only accessed once